### PR TITLE
Added `Root mean squared percentage error` loss function to the `keras.losses`

### PR DESCRIPTION
**Pull request by RakeshJarupula**
Added a new loss/metric function which calculates Root Mean Squared Percentage Error between y_true and y_pred.

### CODE REVIEW

**Comment by RakeshJarupula** [comment]
@googlebot I signed it!

**Review by qlzh727 — CHANGES_REQUESTED** [review]
Please add a unit test for this.

Also add Francois as the API owner since this is introducing a new API.

**Review by fchollet — COMMENTED** [review]
Thanks for the PR. Unfortunately RMSE cannot be added as a loss function because it isn't additive: the sum of the RMSE values on multiple batches isn't equal to the total RMSE for the samples (because of the square root).

If you want to optimize RSME, use MSE as your loss function. You can monitor RMSE as a metric.
